runners/mlcube_docker/mlcube_docker/tests/test_docker_runner.py

```diff
@@ -70,20 +71,22 @@ def test_mlcube_default_entrypoints(self):
         DockerRun(mlcube, task='pwd').run()
 
     @unittest.skipUnless(_HAVE_DOCKER, reason="No docker available.")
+    @patch("io.open", mock_open(read_data=_MLCUBE_CUSTOM_ENTRY_POINTS))
```

Comment:
It's good to keep this pathing feature inside the function (explanation - [772764](https://github.com/mlcommons/mlcube/commit/0d9c4bea562e4816ae027e1c6835339f69d7de1f)):
```python
with patch("io.open", mock_open(read_data=_MLCUBE_CUSTOM_ENTRY_POINTS)):
    mlcube: DictConfig = MLCubeConfig.create_mlcube_config(
        "/some/path/to/mlcube.yaml", runner_config=Config.DEFAULT, runner_cls=DockerRun
    )
```

Reply:
Okay, I have moved this pathing feature back inside the function.